### PR TITLE
updated some wording and typos. Also updated to not hard-code model name

### PR DESCRIPTION
**Pull request by soosung80**

### CODE REVIEW

File: site/en/examples/chat_calculator.ipynb [hunk 1]
```diff
@@ -11,7 +11,7 @@
     },
```
Comment (markmcd) [inline site/en/examples/chat_calculator.ipynb:11]: <p><span style="color: rgba(0, 0, 0, 0.87); letter-spacing: 0.01em;">These steps mention </span><code style="font-weight: 400; letter-spacing: 0.01em;">generate_text</code><span style="color: rgba(0, 0, 0, 0.87); letter-spacing: 0.01em;">, but this is the </span><code style="font-weight: 400; letter-spacing: 0.01em;">chat</code><span style="color: rgba(0, 0, 0, 0.87); letter-spacing: 0.01em;"> notebook.</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

File: site/en/examples/text_calculator.ipynb [hunk 2]
```diff
@@ -11,7 +11,7 @@
     },
```
Comment (markmcd) [inline site/en/examples/text_calculator.ipynb:11]: <p>I'm not sure we ever came to a decision about how to handle model names - run-time selection is a foot-gun, but has some other benefits.</p><p> </p><p>I personally prefer the hard-coded approach, but I think it's out of scope for this review so I filed b/284234353 to track.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Review by markmcd — APPROVED** [review]

File: site/en/examples/chat_calculator.ipynb (hunk 1 above)
Comment (soosung80) [reply site/en/examples/chat_calculator.ipynb:11]: thanks for catching that! fixed in the next commit

File: site/en/examples/text_calculator.ipynb (hunk 2 above)
Comment (soosung80) [reply site/en/examples/text_calculator.ipynb:11]: got it, thanks for opening the bug!

**Review by 8bitmp3 — APPROVED** [review]

**Pull request merged**